app/scripts/lib/createRPCMethodTrackingMiddleware.js

```diff
@@ -310,13 +314,30 @@ export default function createRPCMethodTrackingMiddleware({
         event = eventType.APPROVED;
       }
 
+      let blockaidMetricProps = {};
```

Comment:
This code needs to be code-fenced, which means we need to add code fencing on line 137 as well and also code fence the jsdoc on line 126. The issue is I'm not sure how to code-fence inside jsdoc (or if that's even possible) cc @jpuri @danjm

Reply:
I think it is okay if we don't codefence the jsdoc code. Unless that breaks the build?

Reply:
It breaks the code if we don't code fence it.